Update backup-and-restore.md (#367)
upbound/docs: content/spaces/backup-and-restore.md

        key: creds
```

> Then create two managed control planes with references to the `SharedBackupConfig`.

Reviewer: ```suggestion
Then create two managed control planes.
```

  name: my-awesome-ctp-backup
  namespace: default
spec:
  configRef:
    kind: SharedBackupConfig
    name: default
  controlPlane: my-awesome-ctp
  deletionPolicy: Delete
```

### Restore

<!-- vale off -->
You can restore a control plane's state from a backup in a few steps.
<!-- vale on -->

> First, create a Secret in the source control plane:

Reviewer: I would make it more explicit this is just to have some dummy data to backup and restore, it's not a required step.

### Restore

<!-- vale off -->

> To restore from a backup, create a new control plane with the `sharedBackupConfigRef` or `BackupConfigRef` you created.

Reviewer: there is no sharedBackupConfigRef anymore, this should say only something like:
```
You can restore a controlplane's state from a backup as follows.
```

> To restore from a backup, create a new control plane with the `sharedBackupConfigRef` or `BackupConfigRef` you created.

Author: ```suggestion
You can restore a control plane's state from a backup in a few steps.
```
